content/en/docs/reference/scheduling/policies.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Scheduling Policies
 content_type: concept
-weight: 10
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
```

Comment:
May I know, how this `priority` is to be set?

Reply:
Most pages are 0.9, a few special ones get 1.0

I picked 0.2 because it's a lot less than 0.9 (this is the first low-priority page we've done).

Reply:
Actually, default priority is [0.75](https://github.com/kubernetes/website/blob/d654529ae117d50987c5c5bea7a90e6d2125aa28/config.toml#L88) not 0.9